config/jobs/kubernetes-sigs/jobset/jobset-periodics-main.yaml

```diff
@@ -0,0 +1,190 @@
+periodics:
+  - interval: 12h
+    name: periodic-jobset-test-unit-main
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: jobset
+        base_ref: main
+        path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-jobset-test-unit-main
+      testgrid-num-failures-to-alert: '1'
+      description: "Run periodic jobset unit tests"
+      testgrid-num-columns-recent: '30'
+    labels:
+      preset-dind-enabled: "true"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        env:
+        - name: GO_TEST_FLAGS
+          value: "-race -count 3"
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        args:
+        - make
+        - test
+        securityContext:
+          privileged: true
```

Comment:
any idea why this is necessary?

Reply:
Because you are running Docker in Docker I think it is necessary

Reply:
Gotcha, ok